### ISSUE TITLE
Partial sync gives sponge2 link instead of error messages

### ISSUE THREAD

**Issue by dhalperi** [bazelbuild/intellij#3877]
### Description of the bug:

If I do a partial sync of a file that doesn't compile, my Bazel output shows a sponge2 link instead of the compile errors.

Presumably this is Google-internal behavior that has leaked to the outside world?

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Grab a project, do a partial sync that doesn't compile. I use https://github.com/batfish/batfish. I have not narrowed down a specific repro; it may e.g., involve trying to sync a JUnit test that depends on generated code where the generation script fails.

### Which Intellij IDE are you using? Please provide the specific version.

2022.2.1 (Community Edition)

### What programming languages and tools are you using? Please provide specific versions.

Java. latest rules_jvm_external

### What Bazel plugin version are you using?

`2022.08.09.0.1-api-version-222`

### Have you found anything relevant by searching the web?

No; no issues or PRs mention sponge2 here.

**Comment by dhalperi** [comment]
Full repro:

* check out https://github.com/batfish/batfish@da8ee766aa07384818e46c2182ab7747a9136b93
* sync successfully
    <img width="1666" alt="Screen Shot 2022-08-22 at 11 37 21 AM" src="https://user-images.githubusercontent.com/526415/185994938-95a46a3c-cba3-4a82-a848-ab70408b15ac.png">
* modify `FrrLexer.g4` to remove the `ON_STARTUP: 'on-startup';` line
* Open `FrrGrammarTest.java` and partially sync it.
   <img width="1680" alt="Screen Shot 2022-08-22 at 11 38 22 AM" src="https://user-images.githubusercontent.com/526415/185994972-2fb59d5b-5869-4e7f-a896-b0c11f2a691c.png">

**Comment by mai93** [comment]
I think this is caused by https://github.com/bazelbuild/intellij/commit/0e1479a74e7bfcc73419ad945572ae63d694cb85, I reached out to the author and they are looking into it.

**Comment by dhalperi** [comment]
In case it helps, here's my `~/.bazelrc`

```
common --override_repository=batfish=/Users/dan/git-workspace/batfish

test --test_output=errors
test --build_tests_only=true
test --test_keep_going=false

# Disk cache
# https://docs.bazel.build/versions/master/remote-caching.html#disk-cache
build --disk_cache=/tmp/bazel.cache
build --experimental_guard_against_concurrent_changes
build --experimental_strict_action_env
```

and here's my preferences window

<img width="982" alt="Screen Shot 2022-08-25 at 9 40 19 AM" src="https://user-images.githubusercontent.com/526415/186722029-043f567f-e8b4-47ee-abe1-bf29872cb108.png">

**Comment by mai93** [comment]
This issue should be fixed in the new plugin version (2022.09.07.0.1-api-version-222) currently in [Beta channel ](https://plugins.jetbrains.com/plugin/8609-bazel/versions/beta). You can follow the instructions [here](https://github.com/bazelbuild/intellij#beta-versions) to install it. Please reopen if the issue is still reproducible.
